PyDSS/simulation_input_models.py

```diff
@@ -523,6 +528,11 @@ class HelicsModel(InputsBaseModel):
         default=5,
         alias="Helics logging level",
     )
+    store_intermediate_values: bool = Field(
```

Comment:
@nadiavp @AadilLatif Note this field and default value. Let me know if the default should be flipped.